ajax/libs/sir-trevor/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "sir-trevor",
+  "filename": "sir-trevor.min.js",
+  "description": "Rich content editing entirely re-imagined for the web",
+  "version": "0.5.0",
+  "homepage": "http://madebymany.github.io/sir-trevor-js/",
+  "maintainers": [
+    {
+      "name": "ninjabiscuit",
+      "email": "andy@madebymany.co.uk"
+    },
+    {
+      "name": "zgryw",
+      "email": "lukasz@madebymany.co.uk"
+    },
+    {
+      "name": "raffij",
+      "email": "raffij@gmail.com"
+    }
+  ],
+  "keywords": [
+    "content editing",
+    "rich content editor",
+    "Sir Trevor",
+    "SirTrevor"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/madebymany/sir-trevor-js.git"
+  },
+  "license": "MIT",
+  "npmName": "sir-trevor",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "sir-trevor.+(js|css)",
+        "sir-trevor.min.+(js|css)"
```

Comment:
There is no any conflicts about **be easier** vs **"sir-trevor.test.js" and "sir-trevor.debug.js" and "sir-trevor.test.css" **, and I didn't say to use **"sir-trevor*.+(js|css)"**